toolchains/mxe/Dockerfile.m4

```diff
@@ -80,6 +81,8 @@ mxe_package(fribidi)
 
 mxe_package(glew)
 
+mxe_package(libiconv)
```

Comment:
Why do you add libiconv? We don't use iconv anymore in ScummVM.

Reply:
It's used by `create_sjisfnt` in scummvm-tools.